src/main/java/org/jabref/gui/frame/MainMenu.java

```diff
@@ -146,13 +147,19 @@ private void createMenu() {
         Menu tools = new Menu(Localization.lang("Tools"));
         Menu help = new Menu(Localization.lang("Help"));
 
+        MenuItem saveAllMenuItem = factory.createMenuItem(StandardActions.SAVE_ALL, new SaveAllAction(frame::getLibraryTabs, preferences, dialogService));
+        saveAllMenuItem.disableProperty().bind(Bindings.isEmpty(stateManager.getOpenDatabases()));
+
+        MenuItem newEntryFromPlainTextMenuItem = factory.createMenuItem(StandardActions.NEW_ENTRY_FROM_PLAIN_TEXT, new PlainCitationParserAction(dialogService));
+        newEntryFromPlainTextMenuItem.disableProperty().bind(Bindings.isEmpty(stateManager.getOpenDatabases()));
+        
```

Comment:
Hello @lzmmxh, thank you for your contribution!

Next time, when you're working on any issue, I recommend taking some time to explore the codebase and look for similar patterns related to what you're working on; especially for "good first issues". These issues are designed to help new contributors get familiar with the code.

If you check the codebase, you'll find similar functionality relevant to this issue. We're asking to disable the "Save All" menu item, and you'll see that the "Save" menu item is already disabled as expected. By looking at how the "Save" option is disabled, you can apply the same approach to this task.

I'm not happy with the changes in this pull request, and they will be reverted in PR #11941.